oci/oci.go

```diff
@@ -267,13 +267,14 @@ func (r *Runtime) CreateContainer(c *Container, cgroupParent string) error {
 		logrus.Debugf("Received container pid: %d", ss.si.Pid)
 		if ss.si.Pid == -1 {
 			if ss.si.Message != "" {
-				logrus.Debugf("Container creation error: %s", ss.si.Message)
+				logrus.Errorf("Container creation error: %s", ss.si.Message)
 				return fmt.Errorf("container create failed: %s", ss.si.Message)
 			}
-			logrus.Debugf("Container creation failed")
+			logrus.Errorf("Container creation failed")
 			return fmt.Errorf("container create failed")
 		}
 	case <-time.After(ContainerCreateTimeout):
+		logrus.Errorf("Container creation timeout (%v)", ContainerCreateTimeout)
```

Comment:
We could actually remove all this logrus and just return the error, they don't add much value as the error is returned eventually

Reply:
@runcom I agree the error will be visible somewhere. I added them here because when running with `-log-level warn` I had to look at the kubelet logs to catch this error. CRI-O was just silent about it. I feel such critical errors should appear in the CRI-O log it self, but I'm open.

Reply:
I am fine keeping this in the cri-o logs as it makes it easier to debug.